<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSelectorView</class>
 <widget class="QWidget" name="DataSelectorView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>132</width>
    <height>110</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignHCenter">
    <widget class="PMDButton" name="grayButton">
     <property name="toolTip">
      <string>Hotkey: F1</string>
     </property>
     <property name="text">
      <string>Gray</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="PMDButton" name="distanceButton">
     <property name="toolTip">
      <string>Hotkey: F2</string>
     </property>
     <property name="text">
      <string>Distance</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="PMDButton" name="overlayButton">
     <property name="toolTip">
      <string>Hotkey: F3</string>
     </property>
     <property name="text">
      <string>Overlay</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="PMDButton" name="uniformButton">
     <property name="toolTip">
      <string>Hotkey: F4</string>
     </property>
     <property name="text">
      <string>Gray (Uniform)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PMDButton</class>
   <extends>QPushButton</extends>
   <header>PMDButton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
